frontend/packages/pipelines-plugin/src/components/pipelines/list-page/__tests__/PipelineAugmentRunsWrapper.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { EmptyBox, LoadingBox } from '@console/internal/components/utils';
+import { ListPageWrapper_ as ListPageWrapper } from '@console/internal/components/factory';
+import { PipelineExampleNames, pipelineTestData } from '../../../../test-data/pipeline-data';
+import PipelineAugmentRunsWrapper from '../PipelineAugmentRunsWrapper';
+
+const mockData = pipelineTestData[PipelineExampleNames.WORKSPACE_PIPELINE];
+const { pipeline } = mockData;
+
+type PipelineAugmentRunsWrapperProps = React.ComponentProps<typeof PipelineAugmentRunsWrapper>;
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key) => key }),
+  };
+});
+
+describe('Pipeline Augment Run Wrapper', () => {
+  let pipelineAugmentRunsWrapperProps: PipelineAugmentRunsWrapperProps;
+  let wrapper: ShallowWrapper;
+  beforeEach(() => {
+    pipelineAugmentRunsWrapperProps = {
+      pipeline: {
+        data: [pipeline],
+        loaded: false,
+      },
+    };
+    wrapper = shallow(<PipelineAugmentRunsWrapper {...pipelineAugmentRunsWrapperProps} />);
+  });
+
+  it('Should render loader if data not yet loaded', () => {
+    expect(wrapper.find(LoadingBox).exists()).toBeTruthy();
```

Comment:
Nit, we should always equate booleans to their boolean value. 

```suggestion
    expect(wrapper.find(LoadingBox).exists()).toBe(true);
```

It's not _wrong_ to stay truthy (true is truthy lol), it's just a bit inaccurate on a principle basis.

Reply:
I'm indifferent if this changes before merging. We need a linter to check for `.exists()` and `.tobeTruthy()` on the same expect.